src/utils/facilityData/server/syncFacilityData.ts

```diff
@@ -30,7 +32,8 @@ export async function syncFacilityData({
   const directoryPath = path.join(process.cwd(), "public", "facilityData");
 
   // List all files in the directory
-  const files = await fs.readdir(directoryPath);
+  let files = await fs.readdir(directoryPath);
+  files = _.without(files, '~allFacilities.json');
```

Comment:
We should no longer be placing the individual facilityData files inside of the `/public/` directory because we should not be serving the individual files at all. Instead, we should relocate the files to a `/facilityData/` directory in the root of the project, and only the generated `allFacilities.json` file should appear in `/public/allFacilities` (we could probably remove the `/facilityData` directory from public since `allFacilityData` would be be the only file there anyway).

In the same vein, perhaps removing the unnecessary `~` prefix to the filename, which might suggest its a temporary file when in reality its a generated file intended for long-term use.